src/js/components/products/detail.tsx

```diff
@@ -17,21 +18,19 @@ import {
   // selectProductNotFound,
 } from '@/store/products/selectors';
 
-interface Props {
+type Props = {
   product: Product;
   productSlug: string;
-}
+} & RouteComponentProps;
```

Comment:
@jgerigmeyer  I was thinking this needs to be `RouteChildrenProps` that are defined in  `@types/react-router-dom` but for some reason, but it's saying `@types/react-router-dom"' has no exported member 'RouteChildrenProps'`.  🤷🏽‍♀️